src/GTranslate/LanguageDictionary.cs

```diff
@@ -142,153 +142,167 @@ public bool TryGetLanguage(string code, [MaybeNullWhen(false)] out Language lang
     private readonly IReadOnlyDictionary<string, Language> _languages = new ReadOnlyDictionary<string, Language>(new Dictionary<string, Language>(StringComparer.OrdinalIgnoreCase)
     {
         ["af"] = new("Afrikaans", "Afrikaans", "af", "afr"),
-        ["sq"] = new("Albanian", "Shqip", "sq", "sqi"),
         ["am"] = new("Amharic", "አማርኛ", "am", "amh"),
         ["ar"] = new("Arabic", "العربية", "ar", "ara"),
-        ["hy"] = new("Armenian", "Հայերեն", "hy", "hye"),
+        ["as"] = new("Assamese", "অসমীয়া", "as", "asm", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["ay"] = new("Aymara", "aymar aru [ajmaɾ ˈaɾu]", "ay", "aym", TranslationServices.Google),
         ["az"] = new("Azerbaijani", "Azərbaycan", "az", "aze"),
-        ["eu"] = new("Basque", "Euskara", "eu", "eus"),
+        ["ba"] = new("Bashkir", "Bashkir", "ba", "bak", TranslationServices.Bing | TranslationServices.Microsoft | TranslationServices.Yandex),
         ["be"] = new("Belarusian", "беларуская", "be", "bel", TranslationServices.Google | TranslationServices.Yandex),
+        ["bg"] = new("Bulgarian", "Български", "bg", "bul"),
+        ["bho"] = new("Bhojpuri", "भोजपुरी", "bho", "bho", TranslationServices.Google),
+        ["bm"] = new("Bambara", "ߓߊߡߊߣߊ߲ߞߊ߲", "bm", "bam", TranslationServices.Google),
         ["bn"] = new("Bengali", "বাংলা", "bn", "ben"),
+        ["bo"] = new("Tibetan", "བོད་སྐད་", "bo", "bod", TranslationServices.Bing | TranslationServices.Microsoft),
         ["bs"] = new("Bosnian", "bosanski", "bs", "bos"),
-        ["bg"] = new("Bulgarian", "Български", "bg", "bul"),
-        ["my"] = new("Burmese", "မြန်မာ", "my", "mya"),
         ["ca"] = new("Catalan", "Català", "ca", "cat"),
         ["ceb"] = new("Cebuano", "Binisaya", "ceb", "ceb", TranslationServices.Google | TranslationServices.Yandex),
-        ["ny"] = new("Chichewa", "Nyanja", "ny", "nya", TranslationServices.Google),
-        ["zh-CN"] = new("Chinese (Simplified)", "中文 (简体)", "zh-CN", "zho-CN"),
-        ["zh-TW"] = new("Chinese (Traditional)", "繁體中文 (繁體)", "zh-TW", "zho-TW", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
         ["co"] = new("Corsican", "Corsu", "co", "cos", TranslationServices.Google),
-        ["hr"] = new("Croatian", "Hrvatski", "hr", "hrv"),
         ["cs"] = new("Czech", "Čeština", "cs", "ces"),
+        ["cv"] = new("Chuvash", "Чӑвашла", "cv", "chv", TranslationServices.Yandex),
+        ["cy"] = new("Welsh", "Cymraeg", "cy", "cym"),
         ["da"] = new("Danish", "Dansk", "da", "dan"),
-        ["nl"] = new("Dutch", "Nederlands", "nl", "nld"),
+        ["de"] = new("German", "Deutsch", "de", "deu"),
+        ["dgo"] = new("Dogri", "डोगरी (ḍogrī) ڈوگرى", "dgo", "dgo", TranslationServices.Google),
```

Comment:
The native name should be just "डोगरी".
```suggestion
        ["dgo"] = new("Dogri", "डोगरी", "dgo", "dgo", TranslationServices.Google),
```

Reply:
Fixed.